extensions/skyportal/skyportal/handlers/api/db_stats.py

```diff
@@ -82,116 +91,168 @@ def get(self):
-        data["Newest candidate creation datetime"] = (
-            cand.created_at if cand is not None else None
-        )
-        cand = (
-            DBSession()
-            .query(Candidate)
-            .filter(Candidate.obj_id.notin_(DBSession.query(Source.obj_id)))
-            .order_by(Candidate.created_at)
-            .first()
-        )
-        data["Oldest unsaved candidate creation datetime"] = (
-            cand.created_at if cand is not None else None
-        )
-        data["Latest cron job run times & statuses"] = []
-        cron_job_scripts = DBSession().query(CronJobRun.script).distinct().all()
-        for script in cron_job_scripts:
-            cron_job_run = (
-                CronJobRun.query.filter(CronJobRun.script == script[0])
-                .order_by(CronJobRun.created_at.desc())
-                .first()
-            )
-            data["Latest cron job run times & statuses"].append(
-                {
-                    "summary": f"{script[0]} ran at {cron_job_run.created_at} with exit status {cron_job_run.exit_status}",
-                    "output": cron_job_run.output,
-                }
-            )
 
-        query_tns_count = {
-            "query_type": "count_documents",
-            "query": {
-                "catalog": "TNS",
-                "filter": {},
-            },
-        }
-        response = kowalski.query(query=query_tns_count)
-        data["Number of objects in TNS collection"] = response.get("data")
+        data = {}
 
-        query_tns_latest_object = {
-            "query_type": "find",
-            "query": {
-                "catalog": "TNS",
-                "filter": {},
-                "projection": {"_id": 0, "discovery_date_(ut)": 1},
-            },
-            "kwargs": {"sort": [("discovery_date", -1)], "limit": 1},
-        }
-        response = kowalski.query(query=query_tns_latest_object)
-        response_data = response.get("data", [])
-        latest_tns_object_discovery_date = (
-            response_data[0]["discovery_date_(ut)"] if len(response_data) > 0 else None
-        )
-        data[
-            "Latest object from TNS collection discovery date (UTC)"
-        ] = latest_tns_object_discovery_date
+        # This query is done in raw session as it directly executes
+        # sql, which is unsupported by the self.Session() syntax
+        with DBSession() as session:
+            data["Number of photometry (approx)"] = list(
+                session.execute(
+                    "SELECT reltuples::bigint FROM pg_catalog.pg_class WHERE relname = 'photometry'"
+                )
+            )[0][0]
 
-        for survey in ("PGIR", "ZTF"):
-            utc_now = datetime.datetime.utcnow()
-            jd_start = Time(
-                datetime.datetime(utc_now.year, utc_now.month, utc_now.day)
-            ).jd
-            query_alerts_count = {
+        with self.Session() as session:
```

Comment:
Why not continue with the same session as above?

Reply:
thats a special DBSession() that executes sql directly (same trick we use in Skyportal).

Reply:
right, didn't notice it was a different kind of session.